tests/unit/csi_template_test.go

```diff
@@ -359,6 +393,46 @@ func TestCSITemplateRenderedControllerDeployment(t *testing.T) {
 		name string
 		args args
 	}{
+		{
+			name: "Kubernetes 1.28 with CSI Resizer Enabled",
+			args: args{
+				values: map[string]string{
+					"vCenter.clusterId":                random.UniqueId(),
+					"csiController.csiResizer.enabled": "true",
+				},
+				kubeVersion:       "1.27",
```

Comment:
Test v1.28 for v1.28 tests
```suggestion
				kubeVersion:       "1.28",
```